accounts/templates/accounts/edit.html

```diff
@@ -3,7 +3,11 @@
 
 {% block content %}
 
+{% if form.instance.id %}
```

Comment:
Very minor nitpick but `form.instance.pk` might be better here as that field is guaranteed to exist.